pyproject.toml

```diff
@@ -33,13 +33,14 @@ classifiers = [
     "Topic :: Utilities",
 ]
 dependencies = [
-    "apstools >= 1.7.2",
+    "apstools",
     "bluesky-queueserver-api",
     "bluesky-queueserver",
     "bluesky-widgets",
     "bluesky",
     "caproto",
-    "databroker ==1.2.5",
+    "databroker",
+    "tiled[server]",
```

Comment:
Consider pinning the tiled version. It's still beta, and new releases sometimes break my setup.